war/src/main/less/modules/panes-and-bigtable.less

```diff
@@ -104,6 +104,11 @@ table.bigtable {
     text-align: right;
   }
 }
+.bigtable tfoot th,
+.bigtable .sortbottom th {
+  color: var(--text-color);
+  background-color: var(--white);
```

Comment:
thoughts on just using the application background color?
```suggestion
  background-color: var(--background);
```

Reply:
Sounds reasonable. Maybe even transparent would work